fourkitchens/sous-project
# Update allowed packages for scaffolding

## Purpose:
- Update allowed packages for scaffolding

## Sous PR:
- https://github.com/fourkitchens/sous/pull/7

## Testing:
- Use the sous-project and run composer install on a fresh project, if the project isn't fresh you should be able to run composer update fourkitchens/sous:dev-development-setup
- Verify that the `development.services.yml` file is added to your local codebase with 3 new lines added to disable twig caching.
- This is a laying the foundation PR so that we can add other files to scaffold in the future, this won't work for our module and theme scaffolding though because we can't replace any text within files, just copy them.
